DISCORD_SETUP_GUIDE.md

```
@@ -0,0 +1,198 @@
+
+1. **Start the application:**
+   ```bash
+   mix run --no-halt
+   ```
+
+2. **Verify bot is online:**
+   - Check Discord - your bot should show as online
+   - Check logs for "Successfully registered Discord slash commands"
+
+3. **Test slash commands in Discord:**
+   ```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines
null

(MD031, blanks-around-fences)